### PR TITLE
Added a random overwatch hero picker

### LINKED ISSUE #10: Add overwatch 2 randomizer command

### PR DESCRIPTION
**Pull request by EddinKahvic**

### CODE REVIEW

File: src/commands/games/overwatch.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+      )
+  )
+
+async function execute(interaction: ChatInputCommandInteraction) {
+  const heroes = await getHeroes(interaction.options.getString('role')!)
+
+  if (typeof heroes === 'string') {
+    interaction.reply(heroes)
+    return
+  }
```
Comment (Olivergg1) [inline src/commands/games/overwatch.ts:30]: We should probably find a better way to handle such cases of errors, but fine for now.

File: src/commands/games/overwatch.ts [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+  const index = Math.floor(Math.random() * heroes.length)
+  const result = heroes[index]
```
Comment (Olivergg1) [inline src/commands/games/overwatch.ts]: Rename to `hero`?

File: src/commands/games/overwatch.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  .setDescription('Picks a random hero for you')
+  .addStringOption((option) =>
+    option
+      .setName('role')
+      .setDescription('Pick a role')
+      .setRequired(true)
+      .addChoices(
+        { name: 'damage', value: 'damage' },
+        { name: 'support', value: 'support' },
+        { name: 'tank', value: 'tank' }
```
Comment (Olivergg1) [inline src/commands/games/overwatch.ts:20]: Role names capitalized?

**Review by Olivergg1 — APPROVED** [review]
Looks good, see comments

File: src/commands/games/overwatch.ts (hunk 3 above)
Comment (EddinKahvic) [reply src/commands/games/overwatch.ts:20]: I see lower capitalized options more fit

**Pull request merged**